### PR TITLE
Add validation tests for Java 14 switch expressions

### PR DESCRIPTION
**Pull request by marchof**

### CODE REVIEW

**Comment by Godin** [comment]
@marchof One of the features of switch expressions - is automatic insertion of implicit default clause in case when all enum values are listed. See #771. So I'm wondering if this case should/can be included into this PR?

File: org.jacoco.core.test.validation.java14/src/org/jacoco/core/test/validation/java14/SwitchExpressionsTest.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+/*******************************************************************************
+ * Copyright (c) 2009, 2019 Mountainminds GmbH & Co. KG and Contributors
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Evgeny Mandrikov - initial API and implementation
```
Comment (Godin) [inline org.jacoco.core.test.validation.java14/src/org/jacoco/core/test/validation/java14/SwitchExpressionsTest.java]: @marchof copy-paste? 😉 that's [one of the reasons why I think that `Contributors` field doesn't make sense nowadays](https://github.com/jacoco/jacoco/pull/943#issuecomment-533833196)

Comment (marchof) [reply org.jacoco.core.test.validation.java14/src/org/jacoco/core/test/validation/java14/SwitchExpressionsTest.java]: Grrr, yep let's remove them soon. Fixed.

**Comment by marchof** [comment]
@Godin Good idea, exhaustive example added.

**Pull request merged**